README.md

```diff
@@ -137,7 +137,7 @@ MSVC builds of Rust additionally require an installation of Visual Studio 2017
 
 [Visual Studio]: https://visualstudio.microsoft.com/downloads/
 
-(If you're installing cmake yourself, be careful that “C++ CMake tools for
+(If you're installing CMake yourself, be careful that “C++ CMake tools for
```

Comment:
This works either way, as you're installing a program literally called `cmake` even though the project's name is `CMake`.

Reply:
I made this fix to match the next reference to CMake in this sentence.